src/reducers/article.js

```diff
@@ -0,0 +1,30 @@
+import {
+  CREATE_ARTICLE_SUCCESS,
+  CREATE_ARTICLE_FAILURE,
+  CREATE_ARTICLE_LOADING
+} from "../actionTypes/article";
+import initialState from "../store/initialState";
+
+const article = (state = initialState.article, action) => {
+  switch (action.type) {
+    case CREATE_ARTICLE_LOADING:
+      return {
+        ...state,
+        loading: action.payload
+      };
+    case CREATE_ARTICLE_SUCCESS:
+      return {
+        ...state,
+        article: action.payload
+      };
+    case CREATE_ARTICLE_FAILURE:
+      return {
```

Comment:
Let's try to stick with the naming convention, I will suggest you change this to `CREATE_ARTICLE_ERROR` instead

Reply:
Will work on it